File: model/src/main/java/net/artux/pda/model/profile/NoteModel.java
package net.artux.pda.model.profile;


import java.io.Serializable;
import java.time.Instant;
import java.util.UUID;

import lombok.Data;

@Data
public class NoteModel implements Serializable {

    private UUID id;
    private String title;
    private String content;
    private Instant time;

}

File: model/src/main/java/net/artux/pda/model/profile/NoteModel.kt
package net.artux.pda.model.profile

import java.io.Serializable
import java.time.Instant
import java.util.UUID

data class NoteModel(
    var id: UUID = UUID.randomUUID(),
    var title: String? = null,
    var content: String? = null,
    var time: Instant? = null
) : Serializable